data/pb/commonUiConfig/SearchProjectConfig.json

```diff
@@ -0,0 +1,177 @@
+{
+    "tenantId":"pb",
+    "moduleName":"uiconfigs",
```

Comment:
module name and folder name should be same 
simarly file name and array attribute name should be same.